### PR TITLE
Add simplified Chinese translation

### PR DESCRIPTION
**Pull request by ShuiHuo**

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+msgid "Unable to Complete Recording"
+msgstr "无法完成录音"
+
+#: src/MainWindow.vala:185 src/MainWindow.vala:203
+msgid "The following error message may be helpful:"
+msgstr "下面的错误消息可能会有所帮助："
+
+#: src/StyleSwitcher.vala:24
+msgid "Style:"
+msgstr "样式："
```
Comment (ryonakano) [inline po/zh_CN.po:180]: This looks like the only place where you don't use the "全角 (Quan Jiao)" version of colon.

**Review by ryonakano — COMMENTED** [review]
谢谢你的翻译！
我有一个问题。Just one thing I want to ask, otherwise looking good.

File: po/zh_CN.po (hunk 1 above)
Comment (ShuiHuo) [reply po/zh_CN.po:180]: I don't understand. Can you say it differently?

Comment (ryonakano) [reply po/zh_CN.po:180]: Oh no I found I'd mistaken, sorry! Never mind. :bowing_woman:

**Review by ryonakano — APPROVED** [review]
Sorry for my misreading, looks good! Thank you :heart:

**Pull request merged**